### PR TITLE
Tenant contact info copy to campaign 

### PR DESCRIPTION
**Pull request by bcbeatty**
* Added tables Contact, TenantContact, and FK from Tenant to Location
* Modified Tenant Controler  use Query/Command
* Added new Location Edit and Display Templates 

WIP for #249 

To do: 
* Campaign FK to CampaignContact and Location
* Modify Campaign EditCampaignCommandHandler & CampaignDetailQueryHandler
* Modify UI to copy Tenant Contact and Location

### CODE REVIEW

**Comment by joelhulen** [comment]
At some point, we really need to refactor the project files to eradicate the term "Tenant" for good. I believe its successor is "Organization".

File: AllReadyApp/Web-App/AllReady/Areas/Admin/Controllers/TenantController.cs [hunk 1]
```diff
@@ -1,43 +1,41 @@
...
+        public IActionResult Index()
         {
-            return await Task.Run(() => View(_dataAccess.Tenants));
+            var list = _bus.Send(new TenantListQuery());
+            return View(list);
         }
 
         // GET: Tenant/Details/5
-        public IActionResult Details(System.Int32? id)
+        public IActionResult Details(int? id)
```
Comment (dpaquette) [inline AllReadyApp/Web-App/AllReady/Areas/Admin/Controllers/TenantController.cs]: No need for nullable id here. Change this to int and then we can delete the check for null after.

**Comment by tonysurma** [comment]
@joelhulen - yes not a 'user priority' but a developer and Tony's eyes priority - Tenant should be stricken from all the books :+1: x 1000

**Comment by dpaquette** [comment]
@tonysurma @joelhulen Let's plan a refactoring to rename all classes / properties from Tenant to Organization but _after_ the current milestone?

@tonysurma I suspect the Tenant/Organization crud should be scoped the Site Admins instead of Tenant/Organization admins. Current implementation has it scoped to "TenantAdmin"

**Comment by tonysurma** [comment]
@dpaquette - yes and yes

**Comment by BillWagner** [comment]
@bcbeatty The unit tests in the build are failing. Can you investigate and fix that? Is it the unit test, or a bug in the code?

File: AllReadyApp/Web-App/AllReady/Areas/Admin/Controllers/TenantController.cs [hunk 2]
```diff
@@ -1,43 +1,36 @@
...
-// For more information on enabling MVC for empty projects, visit http://go.microsoft.com/fwlink/?LinkID=397860
+using MediatR;
+using AllReady.Areas.Admin.Features.Tenants;
+using AllReady.Areas.Admin.Models;
 
 namespace AllReady.Areas.Admin.Controllers
 {
-    //[Area("TenantAdmin")]
     [Area("Admin")]
     [Authorize("TenantAdmin")]
```
Comment (dpaquette) [inline AllReadyApp/Web-App/AllReady/Areas/Admin/Controllers/TenantController.cs]: Change this to [Authorize("SiteAdmin")]

**Comment by bcbeatty** [comment]
Code Complete

**Pull request merged**